sdk/app_cpu1/common/drv/timing_manager.c

```diff
@@ -139,14 +146,39 @@ void timing_manager_send_manual_trigger(void)
 }
 
 /*
- * ISR for IRQ_F2P[0:0]. Called when sched_isr in timing
- * manager is set to 1, e.g. when all of the sensors are
- * done and the time has been collected.
+ * Sepcify the interrupt source of the scheduler ISR:
+ *
+ * Mode 0 uses the timing manager's 'trigger' signal, i.e. the control
+ * frequency based on the PWM carrier frequency and the specified user ratio.
+ *
+ * Mode 1 uses the timing manager's 'all_done' signal, calling the scheduler
+ * when all the sensors are done with acquisition. When no sensors are enabled,
+ * the scheduler is called as in mode 0 (based on the trigger). This mode
+ * supports reporting of the timing for each sensor
+ */
+void timing_manager_set_scheduler_source(void)
+{
+    uint32_t config_reg_addr = TIMING_MANAGER_BASE_ADDR + TIMING_MANAGER_ISR_REG_OFFSET;
+#if USER_CONFIG_ISR_SOURCE == 0
+    // Clear slv_reg4[1]
+    Xil_Out32(config_reg_addr, (Xil_In32(config_reg_addr) & 0xFFFFFFFD));
+#elif USER_CONFIG_ISR_SOURCE == 1
+    // Set slv_reg4[1]
+    Xil_Out32(config_reg_addr, (Xil_In32(config_reg_addr) | 0x00000002));
+#endif
+}
```

Comment:
Smart to do this through the `user_config`. Just be sure to mention something in the documentation/user guide so that people know this is how it's done.